src/burn/engine/variable.cpp

```diff
@@ -1897,26 +1897,35 @@ static HRESULT InitializeVariableSystemFolder(
     BOOL f64 = (BOOL)dwpData;
     WCHAR wzSystemFolder[MAX_PATH] = { };
 
-#ifndef _WIN64
```

Comment:
It's not likely to break typical uses and `System64Folder` wasn't documented, so I'm OK with the change but it should probably get highlighted in release notes.